ompi/include/mpif-values.pl

```diff
@@ -242,6 +242,7 @@ sub write_file {
 $constants->{MPI_WIN_DISP_UNIT} = 9;
 $constants->{MPI_WIN_CREATE_FLAVOR} = 10;
 $constants->{MPI_WIN_MODEL} = 11;
+$constants->{MPI_FT} = 12;
```

Comment:
This is definitely problematic.  You can't add anything beyond MPI_FT in our current scheme, because the next thing may be 13 (FT enabled) or may be 12 (FT disabled).